tests/testthat/test-DependencyReporter.R

```diff
@@ -28,24 +28,24 @@ test_that('DependencyReporter structure is as expected', {
                  , ignore.order = TRUE
                  , ignore.case = FALSE
     )
-    
+
     expect_named(object = DependencyReporter$public_fields
                  , expected = NULL
                  , info = "Available public fields for DependencyReporter not as expected."
                  , ignore.order = TRUE
                  , ignore.case = FALSE
     )
-    
+
 })
 
 ### USAGE OF PUBLIC AND PRIVATE METHODS AND FIELDS
 
 test_that('DependencyReporter Methods Work', {
     testObj <- DependencyReporter$new()
-    
+
     # inherited set_package
     expect_silent({
-        
+
```

Comment:
all these whitespace removals are from my default settings in RStudio. lmk if you want me to revert them, IMHO they're not a big deal

Reply:
There are all whitespace additions you are proposing, right?

Reply:
no whitespace removals